scala-package/examples/src/main/java/org/apache/mxnetexamples/javaapi/infer/objectdetector/SSDClassifierExample.java

```diff
@@ -0,0 +1,199 @@
+		CmdLineParser parser  = new CmdLineParser(inst);
+		try {
+			parser.parseArgument(args);
+		} catch (Exception e) {
+			logger.error(e.getMessage(), e);
+			parser.printUsage(System.err);
+			System.exit(1);
+		}
+		
+		String mdprefixDir = inst.modelPathPrefix;
+		String imgPath = inst.inputImagePath;
+		String imgDir = inst.inputImageDir;
+		
+		if (!checkExist(Arrays.asList(mdprefixDir + "-symbol.json", imgDir, imgPath))) {
+			logger.error("Model or input image path does not exist");
+			System.exit(1);
+		}
+		
+		List<Context> context = new ArrayList<Context>();
+		if (System.getenv().containsKey("SCALA_TEST_ON_GPU") &&
+				Integer.valueOf(System.getenv("SCALA_TEST_ON_GPU")) == 1) {
+			context.add(Context.gpu());
+		} else {
+			context.add(Context.cpu());
+		}
+		
+		try {
+			Shape inputShape = new Shape(new int[] {1, 3, 512, 512});
+			Shape outputShape = new Shape(new int[] {1, 6132, 6});
+			
+			
+			int width = inputShape.get(2);
+			int height = inputShape.get(3);
+			String outputStr = "\n";
+			
+			List<List<ObjectDetectorOutput>> output
+					= runObjectDetectionSingle(mdprefixDir, imgPath, context);
+			
+			for (List<ObjectDetectorOutput> ele : output) {
+				for (ObjectDetectorOutput i : ele) {
+					outputStr += "Class: " + i.getClassName() + "\n";
+					outputStr += "Probabilties: " + i.getProbability() + "\n";
+					
+					List<Float> coord = Arrays.asList(i.getXMin() * width,
+							i.getXMax() * height, i.getYMin() * width, i.getYMax() * height);
+					StringBuilder sb = new StringBuilder();
+					for (float c: coord) {
+						sb.append(", ").append(c);
+					}
+					outputStr += "Coord:" + sb.substring(2)+ "\n";
+				}
+			}
+			logger.info(outputStr);
+			
+			List<List<List<ObjectDetectorOutput>>> outputList =
+					runObjectDetectionBatch(mdprefixDir, imgDir, context);
+			
+			outputStr = "\n";
+			int index = 0;
+			for (List<List<ObjectDetectorOutput>> i: outputList) {
+				for (List<ObjectDetectorOutput> j : i) {
+					outputStr += "*** Image " + (index + 1) + "***" + "\n";
+					for (ObjectDetectorOutput k : j) {
+						outputStr += "Class: " + k.getClassName() + "\n";
+						outputStr += "Probabilties: " + k.getProbability() + "\n";
+						List<Float> coord = Arrays.asList(k.getXMin() * width,
+								k.getXMax() * height, k.getYMin() * width, k.getYMax() * height);
+						
+						StringBuilder sb = new StringBuilder();
+						for (float c : coord) {
+							sb.append(", ").append(c);
+						}
+						outputStr += "Coord:" + sb.substring(2) + "\n";
+					}
+					index++;
+				}
+			}
+			logger.info(outputStr);
+			
+		} catch (Exception e) {
+			logger.error(e.getMessage(), e);
+			parser.printUsage(System.err);
+			System.exit(1);
+		}
+		System.exit(0);
+	}
+	
+	static Boolean checkExist(List<String> arr)  {
+		Boolean exist = true;
+		for (String item : arr) {
+			exist = new File(item).exists() && exist;
+			if (!exist) {
+				logger.error("Cannot find: " + item);
```

Comment:
incorrect functionality - the purpose is to log each non-exist item, but with this implement, once one item does not exist, all the following items will be logged as non-exist.

Reply:
This is also wrong in the existing scala example so fixing it there too.